package.json

```diff
@@ -13,7 +13,7 @@
   ],
   "scripts": {
     "pretest": "npm run format && npm run build",
-    "test": "TEST_PORT=1234 TS_NODE_CACHE=0 mocha",
+    "test": "TEST_PORT=1234 TS_NODE_CACHE=0 mocha --exit",
```

Comment:
#582 fixed an issue where the mocha command would never finish. However, as a result, the pull-request became larger, making it difficult to review. In this PR, I would like to take the approach of merging with --exit attached and then excluding --exit later.